src/platform/plugins/shared/data/server/search/session/get_session_status.ts

```diff
@@ -32,7 +33,7 @@ export async function getSessionStatus(
   const searches = Object.values(session.idMapping);
   const searchStatuses = await Promise.all(
     searches.map(async (s) => {
-      const status = await getSearchStatus(deps.internalClient, s.id);
+      const status = await getSearchStatus(deps.internalClient, s.id, s, logger, deps.asUserClient);
```

Comment:
for ES|QL support, quick and dirty and adding a logger

FYI @AlexGPlay @lukasolson